packages/block-editor/src/components/list-view/style.scss

```diff
@@ -211,7 +211,7 @@
 		display: flex;
 		align-items: center;
 		width: 100%;
-		height: auto;
+		height: $button-size-compact;
```

Comment:
Because there are other areas in JS that need to be updated whenever this value is updated, I'm wondering if it'd be better to hard-code this to a specific pixel value? Just thinking that if `$button-size-compact` were to change at any point, it could be easy to miss that certain JS areas also need updating.